### DISCUSSION TITLE
posts/2022/05/simplify-simplify-maybe-for-real-this-time/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#12] [Announcements]
# posts/2022/05/simplify-simplify-maybe-for-real-this-time/

The site enters CTCAJW Mode.

https://www.brycewray.com/posts/2022/05/simplify-simplify-maybe-for-real-this-time/

**Comment by gordon-ansell** [comment]
Just be careful you don't stray into AON territory. AON = "Any Old Nonsense", and that's where my blog's been since it started. My old French teacher was correct when she used to write "lacks focus" on my school reports.

**Reply by brycewray** [reply]
That one I haven’t heard before! 😆
